setup.py

```diff
@@ -6,7 +6,7 @@
     name="pokemon",
 
     # Version number:
-    version="0.34",
+    version="0.35",
```

Comment:
Good job here! Most folks forget to do this.

Reply:
Haha there was reason I called my test module "Clefairy"